.yeoman/templates/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@bbc/psammead-<%= name %>",
+  "version": "0.1.0",
+  "main": "dist/index.js",
```

Comment:
Se should also add  `"module": "esm/index.js",`

Reply:
will look into updating this 👍